From ea78d55036e0d1d23c75878eed7af6845beda55f Mon Sep 17 00:00:00 2001
From: Jason Rogena <j.rogena@cgiar.org>
Date: Thu, 27 Mar 2014 11:36:35 +0300
Subject: [PATCH 130/213] Version 0.8 Revision 6: Removed servicing bull name
 TextEdit from AddCowEvent activity. Users can now enter either the name or
 eat tag number when specifying a servicing bull. Server also updated

---
 MistroFarmer/src/main/AndroidManifest.xml          |  2 +-
 .../cgiar/ilri/mistro/farmer/AddEventActivity.java | 29 +++++++++++-----------
 .../src/main/res/layout/activity_add_event.xml     |  4 +--
 MistroFarmer/src/main/res/values/strings_en.xml    |  3 ++-
 MistroFarmer/src/main/res/values/strings_sw.xml    |  3 ++-
 5 files changed, 22 insertions(+), 19 deletions(-)

diff --git a/MistroFarmer/src/main/AndroidManifest.xml b/MistroFarmer/src/main/AndroidManifest.xml
index 2cbe235..1a9d9b3 100644
--- a/MistroFarmer/src/main/AndroidManifest.xml
+++ b/MistroFarmer/src/main/AndroidManifest.xml
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.cgiar.ilri.mistro.farmer"
     android:versionCode="2"
-    android:versionName="Version 0.8 Revision 5" >
+    android:versionName="Version 0.8 Revision 6" >
 
     <uses-sdk
         android:minSdkVersion="7"
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
index 2bc253b..615a795 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
@@ -72,8 +72,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
     private EditText vetUsedET;
     private TextView bullNameTV;
     private AutoCompleteTextView bullNameACTV;
-    private TextView bullETNTV;
-    private AutoCompleteTextView bullETNACTV;
+    /*private TextView bullETNTV;
+    private AutoCompleteTextView bullETNACTV;*/
     private TextView noOfServicingDaysTV;
     private EditText noOfServicingDaysET;
     private TextView servicingTV;
@@ -124,8 +124,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
         causeOfDeathS = (Spinner)findViewById(R.id.cause_of_death_s);
         bullNameTV = (TextView)findViewById(R.id.bull_name_tv);
         bullNameACTV = (AutoCompleteTextView)findViewById(R.id.bull_name_actv);
-        bullETNTV = (TextView)findViewById(R.id.bull_etn_tv);
-        bullETNACTV = (AutoCompleteTextView)findViewById(R.id.bull_etn_actv);
+        /*bullETNTV = (TextView)findViewById(R.id.bull_etn_tv);
+        bullETNACTV = (AutoCompleteTextView)findViewById(R.id.bull_etn_actv);*/
         noOfServicingDaysTV = (TextView)findViewById(R.id.no_of_servicing_days_tv);
         noOfServicingDaysET = (EditText)findViewById(R.id.no_of_servicing_days_et);
         remarksTV=(TextView)findViewById(R.id.remarks_tv);
@@ -209,8 +209,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
                 noOfServicingDaysET.setVisibility(EditText.GONE);
                 bullNameTV.setVisibility(TextView.VISIBLE);
                 bullNameACTV.setVisibility(AutoCompleteTextView.VISIBLE);
-                bullETNTV.setVisibility(TextView.VISIBLE);
-                bullETNACTV.setVisibility(AutoCompleteTextView.VISIBLE);
+                /*bullETNTV.setVisibility(TextView.VISIBLE);
+                bullETNACTV.setVisibility(AutoCompleteTextView.VISIBLE);*/
                 remarksTV.setVisibility(TextView.GONE);
                 remarksET.setVisibility(EditText.GONE);
             }
@@ -231,8 +231,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
                 eventTypeS.setVisibility(Spinner.GONE);
                 bullNameTV.setVisibility(TextView.GONE);
                 bullNameACTV.setVisibility(AutoCompleteTextView.GONE);
-                bullETNTV.setVisibility(TextView.GONE);
-                bullETNACTV.setVisibility(AutoCompleteTextView.GONE);
+                /*bullETNTV.setVisibility(TextView.GONE);
+                bullETNACTV.setVisibility(AutoCompleteTextView.GONE);*/
                 remarksTV.setVisibility(TextView.GONE);
                 remarksET.setVisibility(EditText.GONE);
             }
@@ -262,7 +262,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
             causeOfDeathS.setAdapter(causesOfDeathArrayAdapter);
         }
         bullNameTV.setText(Locale.getStringInLocale("servicing_bull_name",this));
-        bullETNTV.setText(Locale.getStringInLocale("servicing_bull_ear_tag_number",this));
+        bullNameACTV.setHint(Locale.getStringInLocale("servicing_bull_identifier_hint", this));
+        //bullETNTV.setText(Locale.getStringInLocale("servicing_bull_ear_tag_number",this));
         noOfServicingDaysTV.setText(Locale.getStringInLocale("no_of_days_in_servicing",this));
         remarksTV.setText(Locale.getStringInLocale("remarks",this));
         okayB.setText(Locale.getStringInLocale("okay",this));
@@ -316,8 +317,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
         vetUsedET.setVisibility(EditText.GONE);
         bullNameTV.setVisibility(TextView.GONE);
         bullNameACTV.setVisibility(AutoCompleteTextView.GONE);
-        bullETNTV.setVisibility(TextView.GONE);
-        bullETNACTV.setVisibility(AutoCompleteTextView.GONE);
+        /*bullETNTV.setVisibility(TextView.GONE);
+        bullETNACTV.setVisibility(AutoCompleteTextView.GONE);*/
         remarksTV.setVisibility(TextView.GONE);
         remarksET.setVisibility(EditText.GONE);
         cowIdentifierS.setVisibility(Spinner.GONE);
@@ -369,8 +370,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
             dateTV.setText(Locale.getStringInLocale("date_served", this));
             bullNameTV.setVisibility(TextView.VISIBLE);
             bullNameACTV.setVisibility(AutoCompleteTextView.VISIBLE);
-            bullETNTV.setVisibility(TextView.VISIBLE);
-            bullETNACTV.setVisibility(AutoCompleteTextView.VISIBLE);
+            /*bullETNTV.setVisibility(TextView.VISIBLE);
+            bullETNACTV.setVisibility(AutoCompleteTextView.VISIBLE);*/
             remarksTV.setVisibility(TextView.VISIBLE);
             remarksET.setVisibility(EditText.VISIBLE);
             cowIdentifierS.setVisibility(Spinner.VISIBLE);
@@ -505,7 +506,7 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
                 jsonObject.put("strawNumber", strawNumberET.getText().toString());
                 jsonObject.put("vetUsed", vetUsedET.getText().toString());
                 jsonObject.put("bullName", bullNameACTV.getText().toString());
-                jsonObject.put("bullEarTagNo", bullETNACTV.getText().toString());
+                //jsonObject.put("bullEarTagNo", bullETNACTV.getText().toString());
                 jsonObject.put("noOfServicingDays", noOfServicingDaysET.getText().toString());
                 if(servicingIDs != null) {
                     jsonObject.put("parentEvent", servicingIDs.get(servicingS.getSelectedItemPosition()));
diff --git a/MistroFarmer/src/main/res/layout/activity_add_event.xml b/MistroFarmer/src/main/res/layout/activity_add_event.xml
index 6c66247..bebe4c1 100644
--- a/MistroFarmer/src/main/res/layout/activity_add_event.xml
+++ b/MistroFarmer/src/main/res/layout/activity_add_event.xml
@@ -148,7 +148,7 @@
             android:textSize="@dimen/normal_text_size"
             android:visibility="gone"
             android:textColor="@color/text_input_color"/>
-        <TextView
+        <!--<TextView
             android:id="@+id/bull_etn_tv"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
@@ -163,7 +163,7 @@
             android:layout_marginLeft="@dimen/activity_f_r_edit_text_margin_left"
             android:textSize="@dimen/normal_text_size"
             android:visibility="gone"
-            android:textColor="@color/text_input_color"/>
+            android:textColor="@color/text_input_color"/>-->
         <TextView
             android:id="@+id/no_of_servicing_days_tv"
             android:layout_width="wrap_content"
diff --git a/MistroFarmer/src/main/res/values/strings_en.xml b/MistroFarmer/src/main/res/values/strings_en.xml
index 112c6a6..5fdaef0 100644
--- a/MistroFarmer/src/main/res/values/strings_en.xml
+++ b/MistroFarmer/src/main/res/values/strings_en.xml
@@ -76,7 +76,8 @@
     <string name="specify_en">Specify</string>
     <string name="straw_number_en">Straw Number</string>
     <string name="vet_used_en">Vet Used</string>
-    <string name="servicing_bull_name_en">Name of the Bull used</string>
+    <string name="servicing_bull_name_en">Bull used</string>
+    <string name="servicing_bull_identifier_hint_en">Enter the name, ear tag number or ID of bull</string>
     <string name="servicing_bull_ear_tag_number_en">Ear tag number of the Bull used</string>
     <string name="embryo_number_en">Embryo Number</string>
     <string name="enable_gps_en">Enable GPS</string>
diff --git a/MistroFarmer/src/main/res/values/strings_sw.xml b/MistroFarmer/src/main/res/values/strings_sw.xml
index d455e79..a70cd43 100644
--- a/MistroFarmer/src/main/res/values/strings_sw.xml
+++ b/MistroFarmer/src/main/res/values/strings_sw.xml
@@ -76,7 +76,8 @@
     <string name="specify_sw">Bainisha</string>
     <string name="straw_number_sw">Straw Number</string>
     <string name="vet_used_sw">Daktari aliyetumiwa</string>
-    <string name="servicing_bull_name_sw">Jina la Ndume aliyetumiwa</string>
+    <string name="servicing_bull_name_sw">Ndume aliyetumiwa</string>
+    <string name="servicing_bull_identifier_hint_sw">Ingiza jina, namba ya sikio au kitambulisho (ID) ya ndume</string>
     <string name="servicing_bull_ear_tag_number_sw">Namba ya Sikio ya Ndume aliyetumiwa</string>
     <string name="embryo_number_sw">Embryo Number</string>
     <string name="enable_gps_sw">Wezesha GPS</string>
-- 
2.7.1

